testtools/run.py

```diff
@@ -219,10 +224,18 @@ def _get_runner(self):
         if self.testRunner is None:
             self.testRunner = TestToolsTestRunner
         try:
-            testRunner = self.testRunner(verbosity=self.verbosity,
-                                         failfast=self.failfast,
-                                         buffer=self.buffer,
-                                         stdout=self.stdout)
+            try:
+                testRunner = self.testRunner(verbosity=self.verbosity,
+                                             failfast=self.failfast,
+                                             buffer=self.buffer,
+                                             stdout=self.stdout,
+                                             tb_locals=self.tb_locals)
+            except TypeError:
```

Comment:
Ugh - this nested try / except is starting to look silly. I guess for now it's OK though. Let's hope we don't find a test runner that accepts 'tb_locals' but *not* 'verbosity :D